src/ert/gui/simulation/run_dialog.py

```diff
@@ -674,6 +678,21 @@ def hideEvent(self, event: QHideEvent | None) -> None:
         for file_dialog in self.findChildren(FileDialog):
             file_dialog.close()
 
+    def _show_warning(self, msg: str) -> None:
+        msg_box = QMessageBox(self)
+        msg_box.setObjectName("EnsembleEvaluationWarningBox")
+        msg_box.setIcon(QMessageBox.Icon.Warning)
+        msg_box.setWindowTitle("Ensemble Evaluation Warning")
+        formatted_msg = (
+            "ert has not been able to update the job status for some time. This might "
```

Comment:
This text is very specific to a certain kind of warning. Given that this is a function called for every "EnsembleEvaluationWarning" should not this text be contained in the incoming `warning_message` instead?